meta-ov/recipes-bsp/u-boot/files/0001-video_bmp-implement-BMP-RLE-to-32-bit.patch

```diff
@@ -0,0 +1,160 @@
+From fdad291b7837db232a9a8d071d8d587d8777a485 Mon Sep 17 00:00:00 2001
+From: Max Kellermann <max.kellermann@gmail.com>
+Date: Wed, 9 Feb 2022 17:55:27 +0100
+Subject: [PATCH] video_bmp: implement BMP-RLE to 32 bit
```

Comment:
Are you planning to submit this upstream? It might be a PITA to fix this large patch once it stops applying cleanly.